pep-0696.rst

```diff
@@ -78,15 +79,15 @@ This design pattern is common in projects like:
  - `NumPy <https://github.com/numpy/numpy>`__ - the default for types
    like ``ndarray``'s ``dtype`` would be ``float64``. Currently it's
    ``Unknown`` or ``Any``.
- - `TensorFlow <https://github.com/tensorflow/tensorflow>`__ (this
+ - `TensorFlow <https://github.com/tensorflow/tensorflow>`__ - this
```

Comment:
```suggestion
 - `TensorFlow <https://github.com/tensorflow/tensorflow>`__ — this
```

Typographical nit, but this should be an em dash (you can also do `---` and Sphinx will replace it, if you don't remember the ALT code)